### DISCUSSION TITLE
Contributing to Zed - #10169

### DISCUSSION THREAD

**Discussion by coelhucas** [zed-industries/zed#10187] [Feature Requests]
I opened #10169 and would like to submit a contribution to it myself. I forked the repo, took a look at `CONTRIBUTING.md` and could get some notions of the overall structure, but I'm unsure yet about how `raw_location_settings` and `raw_user_settings` are separated on the codebase in order to give priority to location settings for `deno.enable`.

Initially I've been skimming through `deno.rs`, `language_settings.rs` & `settings_store.rs`. Any guidance is appreciated :)

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
